### PR TITLE
Feature/2394 static pages refactor

### PR DESCRIPTION
**Pull request by richard-jones**

### CODE REVIEW

File: portality/templates/layouts/static_page.html [hunk 1]
```diff
@@ -1,9 +1,23 @@
 {% extends "layouts/base.html" %}
+{% set page = data.frontmatter[page_frag] %}
 
-{% block page_title %}{{ page_title }} &ndash; {% endblock %}
+{% block page_title %}{{ page.title }} &ndash; {% endblock %}
```
Comment (Steven-Eardley) [inline portality/templates/layouts/static_page.html:4]: Should check what's happening with page titles on merge due to https://github.com/DOAJ/doaj/pull/1875

**Review by Steven-Eardley — APPROVED** [review]
Fantastic work.

File: portality/cms/build_templates.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                os.makedirs(os.path.join(out_dir, sub_path))
+            with open(out, "w") as g:
+                g.write(compiled)
+
+    ctx.pop()
+
+    # reset the jinja environment in case we're called in context
+    app.jinja_env.loader = FileSystemLoader(current_paths)
+
+    # TODO: write the output to a temporary directory until we are sure that everything works.  Then at this final stage
```
[the remark this answers is not in the document]
Comment (Steven-Eardley) [reply portality/cms/build_templates.py:65]: We discussed this on tech call, we won't be doing the pre-rendering templates approach, so this comment isn't relevant.

File: portality/templates/layouts/static_page.html (hunk 1 above)
Comment (richard-jones) [reply portality/templates/layouts/static_page.html:4]: Good spot, have carried that forward during the merge

**Comment by richard-jones** [comment]
This PR is now superseded by this one: https://github.com/DOAJ/doaj/pull/1880

**Pull request closed without merging**